test/unit_convex_hull.jl

```diff
@@ -38,23 +38,70 @@ for N in [Float64, Rational{Int}]
     convex_hull!(points_2D) # check in-place version
     @test points_2D == [[N(-1), N(-1)], [N(1), N(0)], [N(1), N(1)], [N(0), N(1)]]
 
-    # three vertex case in 2 dimensions
-    function iscounterclockwise(result, correct_expr)
+    function iscounterclockwise_four_points(result, correct_expr)
```

Comment:
Define `iscounterclockwise` using `any(... circshift(correct_expr, i))` so it applies to any number of points.

Reply:
I think this is a helpful method to define in `src/helper_functions.jl`. You can then include it here:
https://github.com/JuliaReach/LazySets.jl/blob/894fda37e6b0dc2cf9746c1accdebc5d8757f814/test/runtests.jl#L14